src/js/model/validation.js

```diff
@@ -0,0 +1,14 @@
+import {CAR} from "../constants/constants.js";
+
+export const validateCarNames = (namesArr) => {
+  const namesError = namesArr
+    .split(",")
+    .filter((name) => name.trim().length >= CAR.NAME_MAX_LENGTH).length;
```

Comment:
(마이너) 보통 프로젝트 단위가 커지면 이런 조건을 체크하는 함수들의 컨벤션도 통일하고 싶은 니즈가 있는데 저는 이렇게 사용해요!
```suggestion
    .filter((name) => Validator.isLengthOver(name, NAME_MAX_LENGTH).length
```